### DISCUSSION TITLE
[Feature] "View in timeline" feature

### DISCUSSION THREAD

**Discussion by MergenStudios** [immich-app/immich#7433] [Feature Request]
### The feature

When using the amazing smart search feature to find a picture of e.g a trip, I often find myself searching for something I remember from that trip. Smart search usually gives me a result that is exactly what I'm searching for.
But if I want to view the other photos that I took that day, I annoyingly have to remember the date, go the main search page and search for all photos in that date range. 
Now I propose a "view in timeline" button that pops up when you use smart search or view a photo in any context that is not the main timeline. Clicking the button would link to the same image except when using the arrows, you can go forward and backward in time, and not in the album/search results etc., and you are automatically at that point in time in the main gallery if you click the little arrow on the top left

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #4115
